### ISSUE TITLE
Can't install at new nodejs version v7.0.0

### ISSUE THREAD

**Issue by Rukeith** [node-inspector/node-inspector#940]
I try to install node-inspector but it get a error everytime.

<img width="961" alt="screen shot 2016-10-30 at 11 12 44" src="https://cloud.githubusercontent.com/assets/7069605/19834224/def7f6e2-9e91-11e6-9839-cf6991968902.png">
<img width="1179" alt="screen shot 2016-10-30 at 11 12 59" src="https://cloud.githubusercontent.com/assets/7069605/19834225/df321336-9e91-11e6-98e4-82858d09348e.png">

**Comment by OrenElad** [comment]
I got the same errors!!
✗ node -v
v7.0.0
✗ npm -v
3.10.8

**Comment by nfantone** [comment]
Came here for this. Does not work on node `7.0.0`, npm `3.10.9` and, incidentally, yarn.

```sh
Tried to download(404): https://node-inspector.s3.amazonaws.com/debug/v0.7.7/node-v51-darwin-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for v8-debug@0.7.7 and node@7.0.0 (node-v51 ABI) (falling back to source compile with node-gyp)
```

**Comment by leecade** [comment]
+1

**Comment by henriquecustodia** [comment]
+1

**Comment by leecade** [comment]
it seems v8-debug compile failed

**Comment by brandonpsmith** [comment]
+1

**Comment by TDSchreur** [comment]
+1

**Comment by gino8080** [comment]
same problem here
node 7.0.0
npm 3.10.9

**Comment by brandonpsmith** [comment]
Some details may help ...

OSX El Capitan 10.11.6
Node v7.0.0
NPM v3.10.9
Installed via Homebrew

<img width="1318" alt="ss" src="https://cloud.githubusercontent.com/assets/2806503/20065930/7d874fa8-a4d5-11e6-913f-0e804a1e7c24.png">

**Comment by sharikovvladislav** [comment]
Same: https://github.com/node-inspector/node-inspector/issues/944

**Comment by nfantone** [comment]
I might be wrong (I hope I am), but it really looks like this project was left abandoned.

**Comment by brandonpsmith** [comment]
That would be a shame. Are there any good alternatives?

**Comment by nfantone** [comment]
@brandonpsmith Yep. `node --inspect`. See it [in action](http://www.mattzeunert.com/2016/06/01/node-v8-inspector-inspect.html).

**Comment by neligajb** [comment]
+1 , same errors 

OSX 10.12.1
node 7.1.0
npm 3.10.9

**Comment by CocaCola183** [comment]
+1

**Comment by sohailykhan94** [comment]
+1

For the time being downgrading to 6.9.1 works

**Comment by patrikx3** [comment]
also, not working in node 7,

**Comment by patrikx3** [comment]
neither ubuntu, neither windows, neither docker ubuntu

**Comment by pascalopitz** [comment]
+1 same errors, Ubuntu 16.04, node v7.2.1

**Comment by menocomp** [comment]
same in:
node: v7.2.1
npm: 4.0.3

**Comment by anker-leon** [comment]
same errors
macOs 10.12
node: v7.3.0
npm: 3.10.10

**Comment by shikloshi** [comment]
same errors
macOs 10.12
node: v7.3.0
npm: 3.10.10

**Comment by MemoryReload** [comment]
+1

**Comment by brandonpsmith** [comment]
Anyone see the new release v0.7.5 15 days ago? Still not on npm though.

**Comment by nfantone** [comment]
I already mentioned this and I know that it's not a direct replacement (more so if you are using some this via a plugin like `gulp-node-inspector`), but since node `7.0.0` there's no need for `node-inspector` anymore, AFAIK.

See https://github.com/node-inspector/node-inspector/issues/940#issuecomment-259486927

**Comment by pascalopitz** [comment]
--inspect is great for a running process, but my use case is analyzing core dumps. Older ones seem to not load in current chrome devtools, but load okay in node-inspector. I am using nvm to juggle between versions just so I can fire up node-debugger on 6.x

**Comment by brandonpsmith** [comment]
--inspect loads fine in chrome devtools for me. It's just a pain having to click or copy/paste a unique url every time you close your app, make changes, then start it. It slows everything down. If there is some kind of auto reload associated with --inspect I would use it. But AFAIK there is nothing like that. I'm not even sure how that would work with unique url's every time.

**Comment by nfantone** [comment]
Well, `--inspect` is still experimental stuff. So I don't expect it to work in every scenario just yet.

And yes, you are right: the uniqueness of the URL _is_ a drag when reloading the process. You get accustomed  to it quickly, though.

**Comment by smith64fx** [comment]
Any updates on this?

It's not good for the coder's workflow if you have to copy the new URL everytime the application is reloaded 😅

**Comment by brandonpsmith** [comment]
I found this extension to be helpful. https://chrome.google.com/webstore/detail/nodejs-v8-inspector/lfnddfpljnhbneopljflpombpnkfhggl

You simply run your app with the --inspect command and then click the extension button. It will open up the the debugged app with the correct url.

**Comment by hustxiaoc** [comment]
The issue has been resolved, try `node-inspector@1.0.0` please!

**Comment by brandonpsmith** [comment]
Installs for me ... thank you!
